packages/notifi-graphql/lib/NotifiSubscriptionService.ts

```diff
@@ -201,49 +201,57 @@ export class NotifiSubscriptionService {
     });
 
     this._wsClient.on('connecting', () => {
-      console.log('WebSocket connecting'); // TODO: Remove before merge
       this.eventEmitter.emit('wsConnecting');
     });
 
     this._wsClient.on('connected', () => {
-      console.log('WebSocket connected'); // TODO: Remove before merge
-
       this.eventEmitter.emit('wsConnected', this._wsClient!);
     });
 
     this._wsClient.on('closed', (event) => {
-      console.log(`WebSocket closed`); // TODO: Remove before merge
-
       this.eventEmitter.emit('wsClosed', event);
     });
 
     this._wsClient.on('error', (error) => {
-      console.error('WebSocket error:', error); // TODO: Remove before merge
-      if (error instanceof Error) {
-        this.eventEmitter.emit('wsError', error);
+      if (error instanceof Error /*⬅ Client (browser) side error*/) {
+        return this.eventEmitter.emit('wsError', error);
       }
+      this.eventEmitter.emit('wsError', {
+        ...(error as Error),
+        message: 'NotifiEventEmitter: Server side or unknown error',
+      });
```

Comment:
Add this section to emit error while the error comes from server side environments which have different interface of standard Error